### DISCUSSION TITLE
Issues finding sd-auto image in repository

### DISCUSSION THREAD

**Discussion by carlosgmt** [AbdBarho/stable-diffusion-webui-docker#567] [General]
I'm having issues trying to re-deploy this docker image via Portainer.
Can't find the sd-auto (or sd-invoke) image in the repository (docker hub).
What's strange is that it worked in the past (sd-auto v56),  but not now for sd-auto v63. 

Anyone has an idea of what might be happening

**Comment by DevilaN** [comment]
Images are build locally.
First time I am hearing that image has been published on dockerhub for this repository.

**Comment by AbdBarho** [comment]
This container is not published on docker hub due to legal concerns regarding the licenses of the different repositories inside the container.


Can you please provide me the link to the container you are talking about? hopefully it is not something malicious.


In any case, you can just build the container locally and get any version you want by going back in the git history, although not recommended since some stuff won't work from old versions (i.e. extensions or similar).
